neevaco/neeva-ios
# Flip Cheatsheet UGC Indicator Flag

Flip default value to true and add migration

## Checklists

### How was this tested?

### Manual test cases
- delete app data from simulator
- install an older branch
- install new branch
- verify flag

### Screenshots and screen recordings

## Issues addressed
